### PR TITLE
Fix UHS performance-tests target and JS test code

### PR DESCRIPTION
**Pull request by richscott**
- Fix UHS container instantiation via helm
- Change 'disown' shell usage for detaching UHS process to use 'nohup' wrapper prefix instead - 'disown' is available in full Bash, but not minimal /bin/sh versions.
- Fix JS test code to use UHS-style /api/v1/... API prefix, and use correct status_text for parsing response.
- Use recent yunikorn-core (GR fork) image version - this is fixed for now, but we will change it to `latest` once that tag is published for yk-core.

Fixes https://github.com/G-Research/unicorn-history-server/issues/309

### CODE REVIEW

**Review by nikola-jokic — APPROVED** [review]
lgtm

**Review by nikola-jokic — CHANGES_REQUESTED** [review]
Wait, integration tests are failing, can you please fix that

File: Makefile [hunk 1]
```diff
@@ -208,7 +208,7 @@ define start-cluster
 	@echo "**********************************"
 	@echo "Creating cluster"
 	@echo "**********************************"
-	@CLUSTER_NAME=uhs-test $(MAKE) create-cluster
+	$(MAKE) create-cluster
```
Comment (sudiptob2) [inline Makefile:211]: Any specific reason for removing `@CLUSTER_NAME=uhs-test`? Creating a separate cluster for test seemed to be a good idea.

Comment (richscott) [reply Makefile]: Okay - reverted back in commit 70d0db9

**Comment by richscott** [comment]
> Wait, integration tests are failing, can you please fix that

All tests finally work now! Please re-review.

File: Makefile [hunk 2]
```diff
@@ -97,6 +97,10 @@ KIND_VERSION ?= latest
 MINIKUBE ?= $(LOCALBIN_TOOLING)/minikube
 MINIKUBE_VERSION ?= latest
 
+# The release version of Yunikorn images
+# used in integration and performance tests.
+YK_VERSION=fab384e
```
Comment (sudiptob2) [inline Makefile:102]: I just have a small question.
So, we have to change this version every time we release a YK image.
Why don't we release with the `latest` tag?

**Review by sudiptob2 — APPROVED** [review]

**Review by nikola-jokic — APPROVED** [review]

File: Makefile (hunk 2 above)
Comment (dave-gantenbein) [reply Makefile:102]: I always wonder that!

Comment (richscott) [reply Makefile:102]: We will change this to `latest` after we unleash our first `v1.6.0-1-gr0` (or whatever) image release, and @pavlovic-ivan can also make a `latest` tag alias to that.

**Pull request merged**